components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -41,6 +41,61 @@ UpdateVersion.propTypes = {
     releaseNotesUrl: PropTypes.string,
 };
 
+// Label renders a tag showing a name and a description in a tooltip.
+// If a URL is provided, clicking on the tag will open the URL in a new tab.
+export const Label = ({name, description, url, color}) => {
+    const tag = (
+        <span
+            className='tag'
+            style={{backgroundColor: color || ''}}
+        >
+            {name.toUpperCase()}
+        </span>
+    );
+
+    let label;
```

Comment:
should this always default to `tag`? `let label = tag;`?

Reply:
Well, that is a question of style. I find this better to read as it is.

Do you have strong objections with the current style?